docs/primitives/a-text.md

```diff
@@ -14,6 +14,10 @@ Wraps the [text component][text].
 
 ```html
 <a-text value="Hello, World!"></a-text>
+
+<!-- for interactivity add geometry -->
```

Comment:
Good idea. Not sure if interactable is a proper word but it is better than interactive.